NOAA-EMC/EVS
# feature/update_rtofs_ver

<b>Note to developers: You must use this PR template!</b>

## Description of Changes

On 9/13/2024, RTOFS v2.4 was implemented. This updates run.ver for this version change. Closes #545.

## Developer Questions and Checklist
* Is this a high priorty PR? If so, why and is there a date it needs to be merged by?
> Slightly, RTOFS v2.4 was implemented a few days ago.

* Do you have any planned upcoming annual leave/PTO?
> On a half day of work/PTO on 9/16/2024.

* Are there any changes needed for when the jobs are supposed to run?
> No
  
- [x] Jobs contain the approriate file checking and don't run METplus for any missing data.

## Testing Instructions

We could run the RTOFS prep job to make sure it is grabbing the version 2.4 data.
